doc/userguide/upgrade.rst

```diff
@@ -37,6 +37,12 @@ Major changes
 ~~~~~~~~~~~~~
 - Upgrade of PCRE1 to PCRE2. See :ref:`pcre-update-v1-to-v2` for more details.
 
+Security changes
+~~~~~~~~~~~~~~~~
+- Process creation by Suricata is now disabled by default, this can be configured with `security.limit-noproc` (for LUA scripts for instance)
+  This is done through the call of `setrlimit(RLIMIT_NPROC` cf `man setrlimit`.
```

Comment:
typo: missing `)`

Reply:
Well `setrlimit(RLIMIT_NPROC)` is incorrect because setrlimit has 2 arguments...

Will rephrase

Reply:
btw I think `cf` should also be expanded. I took me a while to get used to this acronym, so others may be confused as well.

Reply:
Ok, I thought latin confer was universal ;-)